### DISCUSSION TITLE
Standup (Monday, May 2, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#35] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Research about frontend architecture for #31 

**What are your priorities for the next 24 hours?**:
 - Define architecture with frontend team

**Blockers**:
 -

**Shoutouts**:
 - @CristianCazares @paotg123

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  I didn't do much, i just have been checking on the code sample Carlos did

**What are your priorities for the next 24 hours?**:
 -  Defines tasks with @antonieto and start working on the backend

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19 for helping me installing postgresql

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Read about TypeORM and read @CarlosCRG19 's fake project code.

**What are your priorities for the next 24 hours?**:
 - Define Git guidelines. 
 - Define folder strucure for backend with @BrandonMagana 

**Blockers**:
 - Nothin

**Shoutouts**:
 - @CarlosCRG19 for creating such a great project, will allow us to start development really quick

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Created fake project
 - Read about different technologies on front and back-end

**What are your priorities for the next 24 hours?**:
 - Learn about GithubActions

**Blockers**:
 - None

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Started and almost finished a simple PERN Stack project (a TODO app)
 - Research on #31 
 - Tried to test the "fake_project". Hasn't work so far

**What are your priorities for the next 24 hours?**:
 - Finished the TODO app
 - Tried to go ahead and do #31 with the info so far.

**Blockers**:
 - On the "fake_project" there's a command called "PORT" that is not working.

**Shoutouts**:
 - @CarlosCRG19 for the fake_project
